frontend/apps/reader/modules/readerpaging.lua

```diff
@@ -1019,4 +1019,13 @@ function ReaderPaging:onGotoPrevChapter()
     return true
 end
 
+function ReaderPaging:onToggleReadingOrder()
+    self.inverse_reading_order = not self.inverse_reading_order
+    self:setupTapTouchZones()
+    UIManager:show(Notification:new{
+        text = _((self.inverse_reading_order and "RTL" or "LTR") .. " page direction."),
```

Comment:
Just use the whole string whenever possible. Anything else is bad for localization.

```suggestion
        text = self.inverse_reading_order and _("RTL page direction.") or _("LTR page direction."),
```

As an aside, as written in the PR it wouldn't work regardless, and that'd have to be something like `T(_("%1 page direction"), (self.inverse_reading_order and _("RTL") or _("LTR")))`. But don't do that. ;-)